.github/workflows/pull-request.yml

```
@@ -28,6 +28,12 @@ jobs:
       - name: Run Linter
         run: |
           swiftlint --strict
+          
+      - name: Validate Pull Request Name
+        id: versioning
+        uses: Oliver-Binns/Versioning@main
```

heya - please may we look at pinning the version in this PR
```suggestion
        uses: Oliver-Binns/Versioning@1.0.0
```